libnmstate/nm/ipv6.py

```diff
@@ -60,6 +64,9 @@ def get_info(active_connection):
         if info['dhcp'] or info['autoconf']:
             info['enabled'] = True
             info['address'] = []
+        elif is_link_local_method and _is_ip_config_state(active_connection):
```

Comment:
With `dhcp` and `autoconf` we do not check the ac state, we assume that by the configuration alone, IPv6 is enabled.
Why can't we do the same here?

Reply:
Sure. Will move this to another PR.